<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UFormSalaryTabelSumWidget</class>
 <widget class="QWidget" name="UFormSalaryTabelSumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Підсумковий табель</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_tabelMonthFilter">
       <property name="text">
        <string>Місяць</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_tabelMonthFilter">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="displayFormat">
        <string>MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pidrozdil">
       <property name="text">
        <string>Підрозділ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pidrozdil">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_tabel_sum">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>npr_id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>РОБ.дн</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ЛІК.дн</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ВІДП.дн</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ЗАГ.год.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>НІЧ.год.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>СВ.год.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Прийн.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Звільн.</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
